starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Characters from './Characters';
+
+const CharacterList = props => {
+    return (
+        <div className="char-list">
+
+        {props.starwarsChars.map(charactersMapped => {
```

Comment:
Nice work mapping over the data and generating a component for each character in your data set. Nice use of the name for a key.